x-pack/test/functional/services/transform/transform_table.ts

```diff
@@ -201,6 +230,33 @@ export function TransformTableProvider({ getService }: FtrProviderContext) {
       }
     }
 
+    public async assertTransformRowActionEnabled(
+      action: 'Delete' | 'Start' | 'Stop' | 'Clone' | 'Edit',
+      expectedValue: boolean
+    ) {
+      const selector = `transformAction${action}`;
+      await retry.tryForTime(60 * 1000, async () => {
+        await browser.pressKeys(browser.keys.ESCAPE);
+        await testSubjects.click('euiCollapsedItemActionsButton');
+
+        await testSubjects.existOrFail(selector);
+        const isEnabled = await testSubjects.isEnabled(selector);
+        expect(isEnabled).to.eql(
+          expectedValue,
+          `Expected '${action}' button to be '${expectedValue ? 'enabled' : 'disabled'}' (got '${
+            isEnabled ? 'enabled' : 'disabled'
+          }')`
+        );
+      });
+    }
+
+    public async clickTransformRowActionWithRetry(action: string) {
+      await retry.tryForTime(30 * 1000, async () => {
+        await testSubjects.existOrFail(`transformAction${action}`);
+        await testSubjects.click(`transformAction${action}`);
+      });
```

Comment:
We don't have an assertion here that the click correctly triggered the action, so the retry is limited in detecting a problem. I'd suggest to
- either have individual methods like `clickTransformCloneAction`, `clickTransformStartAction`, ... with individual assertions that the action has been triggered correctly
- or stay with this common one, but then assert that the actions popover is not displayed anymore (that way we know the click was recognized by the popover) and similar to `assertTransformRowActionEnabled` restrict the values for the `action` parameter.

Reply:
Updated here https://github.com/elastic/kibana/pull/92314/commits/c6e7f20fa39c45ab29954a1e62c61f88d160d589